### PR TITLE
Additional vArmour Preview images + Workbook query time range update : to 30 days from various

### PR DESCRIPTION
**Pull request by mocean-v**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - connectorId: vArmourAC
+    dataTypes:
+      - "CommonSecurityLog (vArmour)"
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Collection
+  - LateralMovement
```
Comment (v-jayakal) [inline Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml:16]: Should have minimum of one relevantTechniques. Please check other analytic rules files https://github.com/Azure/Azure-Sentinel/tree/master/Detections/CommonSecurityLog for reference.

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "vArmour"
+  | where DeviceProduct == "AC"
+  | where Activity == "POLICY_VIOLATION"
+  | extend PolicyName = extract("(SMB_REALM_TRAVERSAL\\w+{.*})", 1, DeviceCustomString1)
```
Comment (v-jayakal) [inline Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml:22]: Should have minimum of 1 entity mapping. Please check other analytic rules files https://github.com/Azure/Azure-Sentinel/tree/master/Detections/CommonSecurityLog for reference.

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+id: a36de6c3-3198-4d37-92ae-e19e36712c2e
+name: vArmour AppController - SMB Realm Traversal
+description: |
+  'Detects when SMB traffic crosses Production and Non-Production Realms.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: vArmourAC
+    dataTypes:
+      - "CommonSecurityLog (vArmour)"
```
Comment (v-jayakal) [inline Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml:9]: Just "CommonSecurityLog" is enough for dataTypes, please remove "(vArmour)".

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml (hunk 1 above)
Comment (mocean-v) [reply Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml:16]: Updated to reflect 

relevantTechniques:
  - T1135
  - T1570

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml (hunk 2 above)
Comment (mocean-v) [reply Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml:25]: Added host and IP entities

File: Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml (hunk 3 above)
Comment (mocean-v) [reply Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml]: Updated as requried.

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**